app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
I don't know about that. Your current `id` points to diasporafoundation.org, pinging @denschub about that. An alternative would be to use https://diaspora.github.io/diaspora_federation/schemas.

Also I'm not sure if we want to use a separate versioning for that, or just re-use the same version as the gem?